src/core/webidl.js

```diff
@@ -155,7 +157,9 @@ function defineIdlName(escaped, data, parent) {
   if (showWarnings) {
     const styledName = data.type === "operation" ? `${name}()` : name;
     const ofParent = parentName ? ` \`${parentName}\`'s` : "";
-    const msg = `Missing \`<dfn>\` for${ofParent} \`${styledName}\` ${data.type}. [More info](https://github.com/w3c/respec/wiki/WebIDL-thing-is-not-defined).`;
+    const msg = `Missing \`<dfn>\` for${ofParent} \`${styledName}\` ${
```

Comment:
Maybe you should `npm i` again as AFAIK newer eslint prefers our current styling.

Reply:
trying...

Reply:
Ran `eslint --fix src/core/webidl.js` after update... no change?

Reply:
oh wait... maybe that's my global one.